packages/flutter/lib/src/services/text_input.dart

```diff
@@ -733,6 +747,9 @@ abstract class TextSelectionDelegate {
 
   /// Whether select all is enabled, must not be null.
   bool get selectAllEnabled => true;
+
+  /// The actions that appear in the toolbar, must not be null.
+  List<TextSelectionAction> get actions => <TextSelectionAction>[];
```

Comment:
Given this pattern of using TextSelectionAction, it's not possible to create an action that's not wrapped in FlatButton or CupertinoButton.  Are we ok with that?  Are there any use cases for an action that's not one of those buttons?

My first instinct in seeing TextSelectionAction was to see if it would be possible to just a list of Widgets instead.  However, I see that there's huge value in automatically creating the button for Android or iOS.